docs/reference/msbuild-targets.md

```diff
@@ -113,6 +113,23 @@ The following table describes the MSBuild properties that can be added to a proj
 | `NuspecBasePath` | Base path for the *.nuspec* file. For more information, see [Packing using a .nuspec](#packing-using-a-nuspec-file). |
 | `NuspecProperties` | Semicolon separated list of key=value pairs. For more information, see [Packing using a .nuspec](#packing-using-a-nuspec-file). |
 
+### package item inputs
+
+As described in the section on [including content in your package](#including-content-in-a-package), there is certain [item metadata](/visualstudio/msbuild/msbuild-items#item-metadata) that can be added to [MSBuild items](/visualstudio/msbuild/msbuild-items), which pack will use.
+
+| Metadata | Description |
```

Comment:
I believe the nuspec allows files under `contentFiles/` to set whatever item type they want, for example, you could have `contentFiles/cs/any/stuff.cs` and make it a `<Compile` item, not `<Content`.

@nkolev92 do you remember how to specify this?  I didn't have much of a search yet, so if you don't know, don't waste time looking for it yourself, I'm just hoping you can save me some time 😁

Reply:
BuildAction